addons/crm/models/crm_lead.py

```diff
@@ -389,6 +389,15 @@ def _compute_day_close(self):
             date_close = fields.Datetime.from_string(lead.date_closed)
             lead.day_close = abs((date_close - date_create).days)
 
+    def _get_rotting_depends_fields(self):
+        return super()._get_rotting_depends_fields() + ['won_status', 'type']
+
+    def _get_rotting_domain(self):
+        return super()._get_rotting_domain() + [
```

Comment:
Better combine domains with the new `Domain` class to avoid issues.

This might not be vulnerable here, but this old format is subject to injection as it allows mixing operators (`|`, `&`) and data in the same structure.